public/css/specberus.css

```diff
@@ -82,3 +82,7 @@ td.results a {
     color: #000000;
 }
 
+blockquote {
+  font-size: small;
+  background-color: #ddd;
+}
```

Comment:
This background colour looks a little bit too dark for me to read comfortably; I suggest something like `#f0f0f0`.
Also, I'd add `border: 0;` here to get rid of a little left margin that is visible in this block in a different colour.